arangod/Cluster/Utils/EvenDistribution.cpp

```diff
@@ -74,6 +74,11 @@ Result EvenDistribution::planShardsOnServers(
     return {TRI_ERROR_CLUSTER_INSUFFICIENT_DBSERVERS};
   }
 
+  // Shuffle the servers, such that we don't always start with the same one
+  std::random_device rd;
+  std::mt19937 g(rd());
+  std::shuffle(availableServers.begin(), availableServers.end(), g);
```

Comment:
😲 you are totally right I forgot this step when writing this code!

Thank you for fixing it.